tests/resources/functions/php-special/hello-world.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return function ($request, $response) {
+    echo "log1";
+
+    return $response->json([
```

Comment:
Looks like this function does the same thing as tests/resources/functions/php/index.php

We can remove this file and add just the unicode stuff to the response of tests/resources/functions/php/index.php.

We can then rename this folder to `php-timeout`

Reply:
Removing hello-world.php. Unicode test not needed, all scripts already include emoji in response - should be good enough test.

Folder renamed.